core/roboconf-core/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!--
-   Copyright 2013-2016 Linagora, Université Joseph Fourier, Floralis
+   Copyright 2013-201 Linagora, Université Joseph Fourier, Floralis
```

Comment:
You are right.
I wanted to test whether Checkstyle detected style errors in POM files (it does not). I'll fix it right away. :rotating_light: